### PR TITLE
updated talk_tags to contain more swears

### PR DESCRIPTION
**Pull request by OromisElf**
#### Summary

Content "Add more swear words to the npcs' talking options"

#### Purpose of change

Adds a greater variance in snippets for unhappy NPCs.

#### Describe the solution

There are new entries in the <swear!> and <name_b> categories in the talk_tags.

#### Describe alternatives you've considered

Not adding new swears.

#### Testing

I spawned a bunch of NPCs and let them insult me since they didn't know me yet. Insults that I added were some of them.

### CODE REVIEW

**Comment by Qrox** [comment]
I already hear translators swearing :P.

File: data/json/npcs/talk_tags.json [hunk 1]
```diff
@@ -68,6 +68,75 @@
...
+      "arse",
+      "ass",
+      "assclown",
+      "assface",
+      "asstown",
+      "asswagon",
+      "bitch",
+      "bootlicker",
+      "bugger",
+      "butchery refuse",
```
Comment (topherclay) [inline data/json/npcs/talk_tags.json:81]: ha!

File: data/json/npcs/talk_tags.json [hunk 2]
```diff
@@ -68,6 +68,75 @@
...
+      "coward",
+      "cuck",
+      "cunt",
+      "damn <name_b>",
+      "damned <name_b>",
+      "dildo",
+      "dimwit",
+      "dog",
+      "dumbshit",
+      "dumbster",
```
Comment (Maleclypse) [inline data/json/npcs/talk_tags.json:94]:
```suggestion
      "dumpster",
```

File: data/json/npcs/talk_tags.json [hunk 3]
```diff
@@ -68,6 +68,75 @@
...
+      "dumbster fire",
```
Comment (Maleclypse) [inline data/json/npcs/talk_tags.json:95]:
```suggestion
      "dumpster fire",
```

File: data/json/npcs/talk_tags.json [hunk 4]
```diff
@@ -153,6 +222,21 @@
...
+      "feck",
+      "For serious?",
+      "frigg",
+      "heavens",
+      "heck",
+      "hell",
+      "my luck",
+      "shards and shatters",
+      "the audacity",
+      "what",
```
Comment (Maleclypse) [inline data/json/npcs/talk_tags.json:239]: Maybe also include "the worst!!"

Comment (OromisElf) [reply data/json/npcs/talk_tags.json:240]: thanks for the suggestion :D added it

File: data/json/npcs/talk_tags.json [hunk 5]
```diff
@@ -68,6 +68,75 @@
...
+      "dunderfuck",
+      "effing <name_b>",
+      "emergency ration",
+      "empty headed <name_b>",
+      "expendable <name_b>",
+      "fart",
+      "fat-ass",
+      "frail-ass <name_b>",
+      "frigger",
+      "fungus eater",
```
Comment (ZhilkinSerg) [inline data/json/npcs/talk_tags.json:105]: I think you should not add any content-specific swears- they don't feel right in full conversion mods (which won't have mentioned monsters).

File: data/json/npcs/talk_tags.json [hunk 6]
```diff
@@ -68,6 +68,75 @@
...
+      "goddamn <name_b>",
+      "good for nothing",
+      "inbred",
+      "lazy",
+      "lazy ass",
+      "limp-dick",
+      "low-life",
+      "meat face",
+      "mi-go hugger",
```
Comment (ZhilkinSerg) [inline data/json/npcs/talk_tags.json:114]: Same here.

File: data/json/npcs/talk_tags.json [hunk 7]
```diff
@@ -68,6 +68,75 @@
...
+      "mushy pizza",
+      "pissbrain",
+      "pool noodle",
+      "prick",
+      "pussy",
+      "shitweasel",
+      "slime hugger",
```
Comment (ZhilkinSerg) [inline data/json/npcs/talk_tags.json:121]: Same here.

File: data/json/npcs/talk_tags.json [hunk 8]
```diff
@@ -68,6 +68,75 @@
...
+      "useless",
+      "wanker",
+      "waste",
+      "waste of ammo",
+      "weapon holder",
+      "weiner",
+      "weird <name_b>",
+      "wet sandwich",
+      "wet sock",
+      "z fodder",
```
Comment (ZhilkinSerg) [inline data/json/npcs/talk_tags.json:139]: Same here.

Comment (OromisElf) [reply data/json/npcs/talk_tags.json:139]: sorry for getting back to that only just now but couldn't those conversion mods just also remove those snippets?

Comment (actual-nh) [reply data/json/npcs/talk_tags.json:139]: Perhaps a new `<foe>` to insert as appropriate?

Comment (OromisElf) [reply data/json/npcs/talk_tags.json:139]: could definitely do that if that is wanted

**Pull request merged**